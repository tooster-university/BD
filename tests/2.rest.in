{"open": {"database": "student", "login": "app", "password": "qwerty"}}
{"support": {"timestamp": 74841972, "member": 1, "password": "PW3IUK2", "action": 104, "project": 366, "authority": 362}}
{"trolls": {"timestamp": 97822409}}
{"projects": {"timestamp": 155391590, "member": 1, "password": "PW3IUK2"}}
{"projects": {"timestamp": 208546292, "member": 5, "password": "JXPUFKX"}}
{"votes": {"timestamp": 252986707, "member": 4, "password": "7SHZYQ"}}
{"downvote": {"timestamp": 261699119, "member": 5, "password": "JXPUFKX", "action": 131}}
{"votes": {"timestamp": 305447536, "member": 5, "password": "JXPUFKX"}}
{"actions": {"timestamp": 310740926, "member": 5, "password": "JXPUFKX"}}
{"support": {"timestamp": 332347059, "member": 4, "password": "HCKLJY1", "action": 116, "project": 290, "authority": 387}}
{"upvote": {"timestamp": 348959640, "member": 5, "password": "JXPUFKX", "action": 197}}
{"votes": {"timestamp": 350224213, "member": 4, "password": "HCKLJY1"}}
{"support": {"timestamp": 393407949, "member": 1, "password": "R4AGW1W1", "action": 149, "project": 306, "authority": 404}}
{"actions": {"timestamp": 448652999, "member": 1, "password": "R4AGW1W1"}}
{"support": {"timestamp": 457592694, "member": 1, "password": "R4AGW1W1", "action": 134, "project": 333, "authority": 398}}
{"support": {"timestamp": 491336418, "member": 4, "password": "HCKLJY1", "action": 104, "project": 304, "authority": 457}}
{"protest": {"timestamp": 539220183, "member": 4, "password": "HCKLJY1", "action": 191, "project": 330, "authority": 360}}
