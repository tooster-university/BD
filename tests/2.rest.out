{"status": "OK"}
{"status": "OK"}
{"status": "OK", "data": []}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "OK"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
